docs/First-Installation.md

```diff
@@ -91,11 +91,13 @@ See the following diagrams for reference:
 |:---:|
 |![ps3_controller](https://cloud.githubusercontent.com/assets/10035308/22185413/f10f27de-e099-11e6-97a4-ecbbc82c9e46.png)|
 
+More info at [RetroArch Controller Configuration](RetroArch-Configuration)
+
 ### Hotkey
 
-The Hotkey button enables you to press it in combination with another button to access functions such as saving, loading, and exiting in emulators. It is suggested to use the **Select** button as the hotkey. The following chart shows the default hotkey combinations. For example, if you chose Select as your Hotkey, that means you hold down Select while pressing the other button to execute the command. 
+The Hotkey Enable button enables you to press it in combination with another button to access functions such as saving states, loading states, and exiting back to the EmulationStation main menu. The following chart shows the default hotkey combinations. For example, if you chose Select as your Hotkey Enable button, that means you hold down Select while pressing the other button to execute the command. 
```

Comment:
_exiting back to EmulationStation main menu_ should be _exiting back to EmulationStation_.